Sources/Controllers/2.0/OABaseScrollableHudViewController.h

```diff
@@ -20,6 +20,8 @@ typedef NS_ENUM(NSInteger, EOADraggableMenuState)
 @property (weak, nonatomic) IBOutlet UITableView *tableView;
 @property (weak, nonatomic) IBOutlet UIView *toolBarView;
 @property (weak, nonatomic) IBOutlet UIView *scrollableView;
+@property (weak, nonatomic) IBOutlet NSLayoutConstraint *navbarLeadingConstraint;
```

Comment:
There is no navBarView in BaseScrollableHud, all the logic should be implemented in the subclass and communicated with the base class via inheritance